### PR TITLE
create layout header top (no responsive and background)

### PR DESCRIPTION
**Pull request by nhatanh19**

### CODE REVIEW

File: src/stylesheets/components/_navList.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+@use "../abstracts/" as *;
+
+.nav__item {
+    .nav__item-link {
```
Comment (Hoa0304) [inline src/stylesheets/components/_navList.scss:4]: &__item {

}

File: src/stylesheets/layouts/_header.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    .header-top {
+        width: 100%;
+        height: pxToRem(95);
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        padding: 0 pxToRem(120);
+
+        // logo
+        .header-top__logo {
```
Comment (Hoa0304) [inline src/stylesheets/layouts/_header.scss:15]:
```
&__logo {
    .....
}
```

File: src/stylesheets/layouts/_header.scss [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            .header-top__logo-item {
+                display: flex;
```
Comment (Hoa0304) [inline src/stylesheets/layouts/_header.scss:17]: `
&-item {
    ....
}
`

**Review by Hoa0304 — COMMENTED** [review]
Have you seen the video I sent and the sample repo?

**Comment by nhatanh19** [comment]
> Have you seen the video I sent and the sample repo?

Sorry, I haven't watched the whole video yet

File: src/stylesheets/components/_navList.scss (hunk 1 above)
Comment (nhatanh19) [reply src/stylesheets/components/_navList.scss:4]: > &__item {
> 
> }

i will fix it, thank you

**Pull request closed without merging**